prov/efa/src/rdm/efa_rdm_pke_nonreq.c

```diff
@@ -455,8 +458,14 @@ void efa_rdm_pke_init_write_context(struct efa_rdm_pke *pkt_entry, struct efa_rd
 	rma_context_pkt = (struct efa_rdm_rma_context_pkt *)pkt_entry->wiredata;
 	rma_context_pkt->type = EFA_RDM_RMA_CONTEXT_PKT;
 	rma_context_pkt->version = EFA_RDM_PROTOCOL_VERSION;
-	rma_context_pkt->context_type = EFA_RDM_RDMA_WRITE_CONTEX;
```

Comment:
It is weird that we used to have such typo in wording.

Reply:
Yes, that surprised me too!  😨